src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const calls = 3;
+
+  const getSum = (x) => {
+    return (y) => {
+      count++;
+
+      if (count > calls && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
```

Comment:
you don't need of calls!
```suggestion

  const getSum = (x) => {
    return (y) => {
      count++;

      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }
```